tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -61,6 +61,15 @@ def random_string(length):
     )  # Using only lower case to avoid getting "ERROR" or other string that would be perceived as an error in the log
 
 
+def validate_topic_property(current_value: str, expected_value: str,
+                            property_name: str):
+    if (property_name.endswith(".ms")):
+        # RPK may add "ms" suffix to the value
```

Comment:
bet this is not rpk but a bug of bazel build with which you are testing locally

does this fail in CI? this should't get committed imho

we should get this in https://github.com/redpanda-data/redpanda/pull/25135 instead